.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<!--
+  Thanks for submitting a pull request!
+  We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request.
+
+  Please mark this PR as "Draft" if it is not ready for review.
+-->
+
+### PR Tasks
+
+<!--
+  Before submitting a pull request, please make sure the following is done.
+  If a task does not apply, write [na] instead of checking the box.
+  DO NOT DELETE the PR checklist.
+-->
+
+- [ ] run `yarn` in the repository root
+- [ ] if you've fixed a bug or added code that should be tested, add tests
+- [ ] with the UI running, test if your code breaks existing flows with `yarn e2e:run` at `packages/arb-token-bridge-ui`
+
+#### From `packages/arb-token-bridge-ui` and `packages/token-bridge-sdk`
+
+- [ ] lint your code with eslint, i.e. run `yarn lint`
+- [ ] fix your code with eslint, i.e. run `yarn lintfix`
+- [ ] format your code with `yarn prettier:format`
+- [ ] ensure the unit tests pass with `yarn test`
+  <!-- Tip: `yarn test --watch TestName` is helpful in development. --->
+  <!-- Finally, if you haven't already, complete the CLA. --->
```

Comment:
I'm not sure this should be part of the template. it's nice to have as a checklist for the author, but as a reviewer it doesn't really concern me.

Reply:
do you think it'd be better to be part of the template comment?
but i think people would be trained to ignore the comment text after the first few times opening a PR

Reply:
That's a risk yes, but ultimately it would be covered with CI.
Or maybe group them under `PR follows our contributing guide?`